CIP-1904/README.md

```diff
@@ -0,0 +1,250 @@
+}
+```
+
+This example contains 2 Certificates of Conformity that were both signed using the key pair after multiple key rotations.
+
+Note that the order of signatures in `s` must correspond and match the order of JSON certificates in the off-chain array.
+
+The more [complex examples](./examples/complex) are for type `scm`, but the principles apply to this type in the same manner.
+
+#### Certificate of Conformity revoke
+On occasions where the data in a Certificate of Conformity is incorrect, or the certificate is no longer needed (selling of the goods were cancelled), the authoritative entity should be able to revoke the certificate on-chain.
+
+##### Off-chain format
+The off-chain of a certificate revocation must contain a JSON object with enough identifying information to mark a previoulsy issued certificate as revoked.
+This will be use-case specific as the identifying information must be publicly available.
+
+For example, in the `georgianWine` use-case, the unique identifier of a certificate is not publicly shared but the public attributes `certificate_number` and `certificate_type` together act as a secondary unique identifier.
+
+The format is again an array to allow for batch revocations:
+```
+[{...}, {...}, {...}]
+```
+
+##### On-chain format
+The on-chain format of certificate revocation is identical to the on-chain format for issuing, except the `type` must be `conformityCertRevoke`.
+Please see an example [here](./examples/conformityCertRevoke.json).
+
+### Registered subtypes
+This metadata label may be used for a variety of use-cases, and for this reason, the `st` field may be used to specify a specific use-case.
+This allows 3rd party applications that are only interested in a specific use-case to filter by a particular subtype, but still allow us to maintain a standard across the supply chain space.
+
+Registered subtypes must also provide a clear method to retrieve the off-chain data and verify any public keys are authentic. Embedded public keys must have a clear way to cross-reference for authenticity, and public key URLs must only come from trusted domains for the given subtype.
+
+**Note:** The subtype field is optional! You may choose to omit the subtype in favour of a smaller transaction fee provided there is some other way of retrieving authentic transactions. For example, this might be 1904 labelled transactions that originate from a specific address.
+
+Registered subtypes and their descriptions can be found in [subtypes.json](./subtypes.json).
+
+## Rationale
+The CIP provides a clear way to present verification data for supply chain data records.
+To tackle the possible scale of such a large ecosystem, the specification pays close attention to mechanisms such as batching and key rotation.
+The use of subtypes allows us to capture a broad variety of use-cases and not make opinionated decisions on off-chain data formats.
+
+## Path to Active
+
+### Acceptance Criteria
+<todo>
```

Comment:
Not sure if this needs to be a reference impl. If not, I don't know what to put here other than CF did it so it should be ready to go.